cli/smartcontract/testdata/bindings.yml

```diff
@@ -0,0 +1,297 @@
+package: testdata
```

Comment:
Move it to `cli/smartcontract/testdata/verify.bindings.yml`, otherwise it's not clear what is the original contract. I hope that one day we'll have something like `TestRegenerate` for all our test contracts.